db-client/src/main/java/com/splicemachine/db/client/net/NetConfiguration.java

```diff
@@ -175,6 +175,13 @@
     // Indicates Kerberos authentication.
     public static final int SECMEC_KERSEC = 0x0B;
 
+    // The Plug-in Security Mechanism specifies that an external authentication mechanism is to be
+    // used to carry out the authentication.
+    public static final int SECMEC_PLGIN = 0x0F;
```

Comment:
Even though you are using the PLGIN code, I don't se we are following the DRDA protocol. Quoting from page 53:
> If the plug-in security mechanism (PLGIN) is returned as a SECMEC on the ACCSECRD reply data object, it must be accompanied by the list of supported plug-in modules in the PLGINLST instance variable following the ACCSECRD.

The token should be passed in through the PLGINSECTKN parameter (page 632)

An overview of the flows is described in PLGINOVR (page 627)

Reply:
DRDA protocol has been implemented